### ISSUE TITLE
Skip Tabindex on the header Column

### ISSUE THREAD

**Issue by prsdcse** [react-bootstrap-table/react-bootstrap-table2#1101]
**Is this feature requested before?**
No it is not

**Is your feature request related to a problem? Please describe.**
When we have non actionable column header the WCAG AA complaint is reporting to skip the tabbing on those columns

**Describe the solution you'd like**
Instead hard coding the tabindex to 0 we may get the input through the headerAttrs and make sure it is either 0 or -1

**Describe alternatives you've considered**
No alternatives as of now. Since it is hard coded in the file.

**Comment by AllenFang** [comment]
https://github.com/react-bootstrap-table/react-bootstrap-table2/issues/1099

I will fix it soon

**Comment by AllenFang** [comment]
@prsdcse please upgrade `react-bootstrap-table-next` to newest version, default tabIndex is still 0 but you can configure as -1 via [`column.headerAttrs`](https://react-bootstrap-table.github.io/react-bootstrap-table2/docs/column-props.html#columnheaderattrs-object-function)
